server/export.go

```diff
@@ -18,9 +18,9 @@ import (
 )
 
 const (
-	flagHeight        = "height"
-	flagForZeroHeight = "for-zero-height"
-	flagJailWhitelist = "jail-whitelist"
+	flagHeight           = "height"
+	flagForZeroHeight    = "for-zero-height"
+	flagJailAllowedAddrs = "jail-allowed-addrs"
```

Comment:
actually, can we make these flags public?

Reply:
I'm not what that means. feel free to push to this branch

Reply:
`flag...` -> `Flag...`